aldryn_translation_tools/views.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.http import (
+    Http404,
+    HttpResponseRedirect,
+    HttpResponsePermanentRedirect,
+)
+
+from menus.utils import set_language_changer
+from parler.views import ViewUrlMixin
+
+
+class CanonicalUrlMixin(ViewUrlMixin):
+    #
+    # Provides configurable control over how non-canonical URLs to views are
+    # handled. A view can specify by setting the property
+    # 'non_canonical_url_response_type' to 200, 301, 302 or 404. By default,
+    # handling will be to temporarily redirect to the canonical URL.
+    #
+    non_canonical_url_response_type = 302
+
+    def get_non_canonical_url_response_type(self, request, *args, **kwargs):
+        """
+        A getter for the view property non_canonical_url_response_type.
+        Override as required.
+        """
+        response_type = getattr(self, "non_canonical_url_response_type", None)
```

Comment:
yep I rather follow more strict guidelines in wether I use either single or double quotes